src/compiler/emitter.ts

```diff
@@ -28,7 +28,7 @@ import {
     BundleFileTextLikeKind,
     CallExpression,
     CallSignatureDeclaration,
-    CaseBlock,
+    canHaveLocals, CaseBlock,

```

Comment:
There's no lint rule for this, but these should be columns.

Hopefully one day we can get formatting automated...

Reply:
One day...
![futuristic city](https://user-images.githubusercontent.com/9800850/207276227-45df5936-0e4a-4b94-aca6-c10fafd490e5.png)

🤣 (sorry, I couldn't resist)

Reply:
> There's no lint rule for this, but these should be columns.
> 
> Hopefully one day we can get formatting automated...

I've also noticed that Organize Imports seems to be sorting `__String` to the end rather than the start.

Reply:
Yeah, that's: https://github.com/microsoft/TypeScript/pull/51733